src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,30 @@
+.card__stars {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  font-weight: 400;
+  font-size: 10px;
+  margin-top: 16px;
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
```

Comment:
It's better to add 16px to variables.

Also you can create `@mixin` for square and replace `width` and `height` here with `@include square($star-size)`